README.rst

```diff
@@ -36,109 +36,96 @@ your Python code to a file.
+   from ev3dev2.sensor.lego import TouchSensor
+   from ev3dev2.led import Leds
 
-Required: Import the library
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   # TODO: Add code here
 
-If you are using an EV3 brick (which is the case for most users), add the
-following to the top of your file:
+The first line should be included in every Python program you write
+for ev3dev. It allows you to run this program from Brickman, the graphical
+menu that you see on the device screen. You will need to add additional classes
+to the import list if you want to use other types of devices.
 
-.. code-block:: python
+You should use the ``.py`` extension for your file, e.g. ``my-file.py``.
 
-  import ev3dev.ev3 as ev3
+Important: Make your script executable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you are using a BrickPi, use this line:
+To be able to run your Python file, **your program must be executable**. If
+you're using the `ev3dev Visual Studio Code extension`_, this step will be
+automatically performed when you download your code to the brick. **You can
+skip this step.**
 
-.. code-block:: python
+**To mark a program as executable from the command line (often an SSH session),
+run **``chmod +x my-file.py``.
 
-  import ev3dev.brickpi as ev3
+You can now run ``my-file.py`` via the Brickman File Browser or you can run it
+from the command line by preceding the file name with ``./``: ``$ ./my-file.py``
 
 Controlling the LEDs with a touch sensor
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This code will turn the left LED red whenever the touch sensor is pressed, and
-back to green when it's released. Plug a touch sensor into any sensor port and
-then paste in this code - you'll need to hit ``Enter`` after pasting to complete
-the loop and start the program.  Hit ``Ctrl-C`` to exit the loop.
+back to green when it's released. Plug a touch sensor into any sensor port before
+trying this out.
 
 .. code-block:: python
 
-  ts = ev3.TouchSensor()
+  ts = TouchSensor()
+  leds = Leds()
+
   while True:
-      ev3.Leds.set_color(ev3.Leds.LEFT, (ev3.Leds.GREEN, ev3.Leds.RED)[ts.value()])
+      if ts.is_pressed:
+          leds.set_color(leds.led_groups.LEFT, leds.led_colors.GREEN)
+      else:
+          leds.set_color(leds.led_groups.LEFT, leds.led_colors.RED)
 
 Running a motor
 ~~~~~~~~~~~~~~~
 
-Now plug a motor into the ``A`` port and paste this code into the Python prompt.
-This little program will run the motor at 500 ticks per second, which on the EV3
-"large" motors equates to around 1.4 rotations per second, for three seconds
-(3000 milliseconds).
+This will run a LEGO Large Motor at 75% power for 5 rotations.
 
 .. code-block:: python
 
-  m = ev3.LargeMotor('outA')
-  m.run_timed(time_sp=3000, speed_sp=500)
-
-The units for ``speed_sp`` that you see above are in "tacho ticks" per second.
-On the large EV3 motor, these equate to one tick per degree, so this is 500
-degress per second.
-
-
+  m = LargeMotor(OUTPUT_A)
+  m.on_for_rotations(75, 5)
```

Comment:
This is offtopic, but looking at this line of code I feel it could be more readable. Is 75 here power or rotations? Why the power attribute is on the first place when the method is called `on_for_rotations`? Why not `m.on_for_rotations(5, power=75).`

Reply:
We can use the new methodology from #407